<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>расчёт корней квадратного уравненя</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_1_a">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>30</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_x2">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;x&lt;span style=&quot; vertical-align:super;&quot;&gt;2  &lt;/span&gt;+&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_1_b">
     <property name="geometry">
      <rect>
       <x>56</x>
       <y>30</y>
       <width>30</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_x">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>30</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>x +</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_1_c">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>30</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_0">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>=0</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="ButtonQuEducation1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>расчитать x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_qu_education">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>313</x>
      <y>0</y>
      <width>20</width>
      <height>219</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>281</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>расчёт стороны треугольника</string>
    </property>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>118</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>3</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_triangleA">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>40</y>
       <width>30</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_triangleB">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>30</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_triangle_angle">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>30</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_triangleA">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>40</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>а</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_triangleB">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>90</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>b</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_triangle_angle">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>50</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>угол</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioDegrees">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>градусы</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioRadians">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>45</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>радианы</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Button2Triangle">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>расчитать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_triangle">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditCopy">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>671</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3 задание</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditPaste">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>260</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Button3HTML">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>html</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button3Replace">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>310</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>заменить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button3Add">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>добавить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>674</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
